frontend/cypress/integration/common/namespace.ts

```diff
@@ -0,0 +1,7 @@
+import { And } from 'cypress-cucumber-preprocessor/steps';
+
+And('user selects the {string} namespace in the NamespaceSelector', (namespace: string) => {
+  cy.getBySel('namespace-dropdown').click();
```

Comment:
Not a blocker, but not sure if I'd prefer these "sugar" functions for things that are more elaborated, rather than a simply selector helper.

![image](https://user-images.githubusercontent.com/1662329/165265326-3451d92a-0c9d-4fd3-877b-ab637f5f9c50.png)

Personally, to me, it helps me when I read the full selector.

But I see interesting these commands for something more elaborated, like the pattern we have to "click-and-wait", those repeated elements perhaps can be implemented under the commands.

Again, not a blocker, we an iterate on this in the future. These tests will have an active maintenance.

Reply:
That's fair. I was mostly following the pattern from the docs/sample app which include this. I think there's some value in not having to get the `data-test` selector right each time you want to use it but it's a very simple command.